pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -175,4 +176,28 @@ in {
 
     spl = splUnstable;
   };
+
+  zfsCryptoStability = common {
```

Comment:
This should be `zfsUnstable`. The other version should be called `zfsLegacyCrypto`.

Reply:
I have switched between unstable and CryptoStability (now LegacyCrypto)